lib/OpenQA/WebAPI/Controller/API/V1/JobSettings.pm

```diff
@@ -15,31 +15,47 @@ Filters jobs based on a single setting key/value pair.
 
 =item C<key>
 
-The setting key to filter by. It accepts a string or a glob of a job variable.
-Some variables are not stored and they can not be used.
+The setting key to filter by. Some variables are not stored and they can not be
+used.
+
+=item C<value>
+
+Returns all jobs where the value of C<key> is C<value>. This can be a string
+without special characters.
 
 =item C<list_value>
 
-The value to match for the given key. This can be a string without special characters.
+Returns all jobs where the value of C<key> contains a comma-separated list that
+in turn contains C<list_value>. This can be a string without special characters.
```

Comment:
Why without special characters?

Reply:
This sentence was present before. I only changed the description to account for the new `value` parameter.